imap_processing/io.py

```diff
@@ -0,0 +1,162 @@
+"""Input/output capabilities for the IMAP data processing pipeline."""
```

Comment:
Is this meant to replace the sds-api.py script?

Reply:
Great question. Currently, it is just functions to help out during the processing. We could go one of two ways. Pull this content out and put it over into the `sds-api.py` script, and depend on that here. I sort of avoided that so we didn't have to worry about releasing versions and syncing things like that between the two repositories. The other would be to add this to our cli or make a new one that could be installed alongside the package.

The major con to turning this into a cli is that a user has to bring along all the `imap_processing` dependencies, whereas the `sds-api.py` script/helper repo could be slimmer and not require that.

Thoughts/preferences on this?

Reply:
What syncing would we need to do between the two repositories if we moved this to sds-api? Wouldn't that have its own versioning and be a separate module that we import into imap_processing?

Reply:
syncing releases. So we'd need to make a pypi release of that and this at the same time so they could depend on the right versions of each other.